auth/client/iam/interface.go

```diff
@@ -50,6 +51,7 @@ type Client interface {
 	OpenIdCredentialIssuerMetadata(ctx context.Context, webDID did.DID) (*oauth.OpenIDCredentialIssuerMetadata, error)
 
 	VerifiableCredentials(ctx context.Context, credentialEndpoint string, accessToken string, proofJWT string) (*CredentialResponse, error)
-	// RequestObject is returned from the authorization request's 'request_uri' defined in RFC9101.
-	RequestObject(ctx context.Context, requestURI string) (string, error)
+	// RequestObject is returned from the authorization request's 'request_uri' endpoint defined in RFC9101
```

Comment:
format shouyld be `XXX <verb> <description>`, so something like `RequestObject retrieves the OAuth2 authorization Request Object bladiebla`